snapcraft/plugins/catkin.py

```diff
@@ -105,6 +105,7 @@ def env(self, root):
                 root,
                 self.options.rosversion),
             'ROS_MASTER_URI=http://localhost:11311',
```

Comment:
Ah, okay. It is actually configurable, I just need to figure out how to do that with Snappy, which I'll make a different PR.